internal/resourcepolicies/resource_policies.go

```diff
@@ -0,0 +1,143 @@
+package resourcepolicies
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/pkg/errors"
+	v1 "k8s.io/api/core/v1"
+)
+
+type VolumeActionType string
+
+const Skip VolumeActionType = "skip"
+
+// Action defined as one action for a specific way of backup
+type Action struct {
+	// Type defined specific type of action, currently only support 'skip'
+	Type VolumeActionType `yaml:"type"`
+	// Parameters defined map of parameters when executing a specific action
+	Parameters map[string]interface{} `yaml:"parameters,omitempty"`
+}
+
+// VolumePolicy defined policy to conditions to match Volumes and related action to handle matched Volumes
+type VolumePolicy struct {
+	// Conditions defined list of conditions to match Volumes
+	Conditions map[string]interface{} `yaml:"conditions"`
+	Action     Action                 `yaml:"action"`
+}
+
+// currently only support configmap type of resource config
+const ConfigmapRefType string = "configmap"
+
+// ResourcePolicies currently defined slice of volume policies to handle backup
+type ResourcePolicies struct {
```

Comment:
The `ResourcePolicies` is only used internally, so let's change it to an unexported type

Reply:
updated